secret-provider/src/main/scala/io/lenses/connect/secrets/config/AWSProviderConfig.scala

```diff
@@ -21,6 +21,7 @@ object AWSProviderConfig {
   val AWS_SECRET_KEY:    String = "aws.secret.key"
   val AUTH_METHOD:       String = "aws.auth.method"
   val ENDPOINT_OVERRIDE: String = "aws.endpoint.override"
+  val AWS_CROSS_ACCOUNT_REGION:        String = "aws.cross.account.region"
```

Comment:
I general I'm not sure I understand the need for the alt.region property as 

* it is always used wherever region is used and simply overrides it - I think this can be removed in favour of just changing the existing region property in connector configuration?

* the condition that activates the new configuration is actually only activated based on the presence of an $ in the secret id, so I don't think any configuration is necessary here.